### DISCUSSION TITLE
反腐败永无止境

### DISCUSSION THREAD

**Discussion by SuDaQi** [WeNeedHome/SummaryOfLoanSuspension#497] [General]
不得罪成百上千的腐败分子，就要得罪十三亿人民。——习近平

**Comment by k0spider** [comment]
谁指使你说实话的，你有什么意图。你这是恶意说话

**Comment by ChinaChenMingQuan** [comment]
说句你们常说的话，你不贪，有的是人来贪！

**Comment by liuwenwu520** [comment]
政府官员资产不透明必然会导致贪污腐败

**Comment by mccken** [comment]
已经有丰富的历史经验告诉我们，像苏联那样子，走社会主义道路，由共产党统治，就一定会有贪污，这是底层逻辑

**Reply by zcaaron** [reply]
赞同

**Reply by Crazy-Creeper** [reply]
任何体制都有贪污，不过你说的对，共产党这种自上而下的人治造成的贪污绝对是让人匪夷所思的

**Reply by Go-loneliness-of-man** [reply]
说的好像美国就没贪污了？哪国没贪污？什么脑子

**Comment by zcaaron** [comment]
没人监督啊。在一个这样没人监督的环境人，谁都会贪。

**Comment by wedxz** [comment]
每个爬上去的人都废了九牛二虎之力,你说不贪?     那之前的努力不白费了?

**Comment by hktk1024** [comment]
家丑不可外扬🐶

**Comment by rechardlc** [comment]
绝对的权力就有绝对的贪污~这是一个必然关系

**Comment by shawn-yee** [comment]
肯定不是全部啦
底层还有好多马仔还没混到有油水的级别
一旦升上去了，嘿嘿

**Comment by dongyc519** [comment]
现在daguo的系统性问题是源代码级别的bug，必须重构才能解决。

**Comment by yuhaoboris** [comment]
問這種問題，你是第一天當中國人嗎？

**Comment by sthfake** [comment]
说话要讲事实，咱们读明朝那些事，海瑞，于谦都不是贪官

**Reply by zcaaron** [reply]
以前还有儒家思想约束一下，现在是啥都没有

**Comment by liuwenwu520** [comment]
不管是在哪个国家，只要违法的成本低，比如不易发现、轻罚等，就会造成贪污腐败盛行

**Comment by a984120978** [comment]
因为我国民间自古以来就是  升官发财

**Comment by MYWmiss** [comment]
不怕贪，就怕没能力

**Reply by a984120978** [reply]
我要是当官我也探

**Comment by lettermessenger** [comment]
没有不偷腥的猫

**Comment by kipshi** [comment]
反腐要亡党，不反腐要亡国

**Comment by TALK1949** [comment]
围着粪坑打苍蝇😓

**Comment by Lilihx** [comment]
权利太大。政府应当是为人民工作的，不是作威作福的
